core/tooltip.js

```diff
@@ -111,6 +111,45 @@ Blockly.Tooltip.MARGINS = 5;
  */
 Blockly.Tooltip.DIV = null;
 
+/**
+ * Returns the tooltip text for the given element.
+ * @param {?Object} object The object to get the the tooltip text of.
```

Comment:
I kind of wanted to leave it as open as possible. This way you can pass it any object without a tooltip property (eg `{}`) and it will return `''`' as the tooltip.